data/filetypes.common

```diff
@@ -106,6 +106,7 @@ error=0xff0000;0xBFBFBF;false;italic
 
 selection=0x000000;0xc0c0c0;false;true
 current_line=0x000000;0xf0f0f0;true;
+current_word=0xffffff
```

Comment:
I tried all kinds of combinations, nothing seemed to work, so this is where I left it :)